core/src/test/java/com/linecorp/armeria/common/HttpHeadersBaseTest.java

```diff
@@ -120,6 +120,20 @@ void testGetBooleanOperation() {
         headers.add("baz", "false");
         assertThat(headers.containsBoolean("baz", false)).isTrue();
         assertThat(headers.containsBoolean("baz", true)).isFalse();
+
+        headers.add("dup1", "v1");
+        headers.add("dup1", "v2");
+        headers.add("dup1", "true");
+        headers.add("dup1", "false");
+        assertThat(headers.containsBoolean("dup1", true)).isTrue();
+         assertThat(headers.containsBoolean("dup1", false)).isTrue();
+
+        headers.add("dup2", "v1");
+        headers.add("dup2", "v2");
+        headers.add("dup2", "1");
+        headers.add("dup2", "0");
+        assertThat(headers.containsBoolean("dup2", true)).isTrue();
+        assertThat(headers.containsBoolean("dup2", false)).isTrue();
```

Comment:
```java
        headers.add("dup1", "v1");
        headers.add("dup1", "v2");
        headers.add("dup1", "true");

// Actual: containsBoolean -> getBoolean("dup1") -> "v1" -> contains: false
// Expected: containsBoolean -> getBoolean("dup1") -> "true" -> contains: true
        assertThat(headers.containsBoolean("dup1", true)).isTrue(); 
```
With current `containsBoolean`, above test case failed cause it uses `getBoolean` inside!